docs/reference/api/README.md

```diff
@@ -0,0 +1,9 @@
+@wpengine/headless / [Exports](modules.md)
+
+# WordPress Headless Framework
+
+[![Version](https://img.shields.io/npm/v/@wpengine/headless.svg)](https://npmjs.org/package/@wpengine/headless)
+
+NOTE: This project is in the early stages of development, but it does contain useful functionallity for headless WordPress sites like authentication and post previews. Be sure to install the [WordPress plugin](https://github.com/wpengine/headless-framework) that enables the functionality in this package.
```

Comment:
Just noticed one instance of `functionality` is misspelled.

```suggestion
NOTE: This project is in the early stages of development, but it does contain useful functionality for headless WordPress sites like authentication and post previews. Be sure to install the [WordPress plugin](https://github.com/wpengine/headless-framework) that enables the functionality in this package.
```

Reply:
Thanks for spotting it. Fixed in https://github.com/wpengine/headless-framework/pull/132/commits/72420dfe1bb19ab330a40093f5a86dd27190211c (another file was affected).